corporate/views/support.py

```diff
@@ -556,6 +553,7 @@ def remote_servers_support(
             support_view_request = SupportViewRequest(
                 support_type=SupportType.configure_fixed_price_plan,
                 fixed_price=fixed_price,
+                sent_invoice_id=sent_invoice_id,
```

Comment:
Does this end up supporting invoices in both full Stripe URL and ID format? We might want both to work?

Reply:
Oh, yeah, that's a good idea. We just need to extract the ID from the URL.

Reply:
[CZO Discussion](https://chat.zulip.org/#narrow/stream/191-kandra-support/topic/invoice-based.20billing/near/1731749)

We make an API call to fetch payments-page URL on /upgrade. API call is made when a customer tries to visit /upgrade after fixed-price offer is configured with sent_stripe_id, so this is going to be rare.